packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -108,6 +140,17 @@ class WebFlutterDriver extends FlutterDriver {
     throw UnimplementedError();
   }
 
+  void _logCommunication(String message) {
+    if (_printCommunication) {
+      driverLog('WebFlutterDriver', message);
+    }
+    if (_logCommunicationToFile) {
+      final File file = fs.file(path.join(testOutputsDirectory, 'flutter_driver_commands_$_driverId.log'));
+      file.createSync(recursive: true); // no-op if file exists
+      file.writeAsStringSync('${DateTime.now()} $message\n', mode: FileMode.append, flush: true);
```

Comment:
You don't need `flush: true` since you're not trying to immediately read it back

Reply:
I copied from `VMServiceFlutterDriver`. but if not need `flush: true` i will remove for both.